atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -197,11 +204,24 @@
 public abstract class TransactionManagers {
     private static final int LOGGING_INTERVAL = 60;
     private static final Logger log = LoggerFactory.getLogger(TransactionManagers.class);
+    private static final Duration REFRESH_INTERVAL = Duration.ofSeconds(1);
 
     public static final LockClient LOCK_CLIENT = LockClient.of("atlas instance");
 
     abstract AtlasDbConfig config();
 
+    @Value.Default
+    Refreshable<Optional<AtlasDbRuntimeConfig>> runtimeConfig() {
```

Comment:
So this is great, for internal ski product that creates a lot of TransactionManagers I can pass this directly and hence not create a threadpool per TransactionManager.

But if it's not passed in, you need to make sure that this threadpool is cleaned up. TransactionManagers have a #close method, so I think you want to create this threadpool somewhere further down where it could be cleaned up.